docs/crud/crud.md

```diff
@@ -54,5 +54,14 @@ The names of the secrets are derived as follows: `<cluster-name>-<cluster-hash>-
 Example of how to decode a kubeconfig from secret:
 
 ```sh
-kubectl get secrets -n claudie <cluster-name>-<cluster-hash>-kubeconfig -o jsonpath='{.data.kubeconfig}' | base64 -d > your_kubeconfig.yaml
+kubectl get secrets -n claudie -l claudie.io/cluster=<cluster name> claudie.io/project=<project name> claudie.io/output=kubeconfig -o jsonpath='{.data.kubeconfig}' | base64 -d > your_kubeconfig.yaml
```

Comment:
We can remove the variability here by using only the `claudie.io/output=kubeconfig` label, which is perfectly sufficient if there's only one cluster.

This will be simple enough for people new to K8s, but still understandable for veterans.

Not sure though, what do you think ?

Reply:
Yeah, we can do that :)

Reply:
Yeah, we can do that :)